crates/matrix-sdk-base/CHANGELOG.md

```diff
@@ -2,24 +2,80 @@
 
 All notable changes to this project will be documented in this file.
 
-# unreleased
+## [0.8.0] - 2024-11-19
 
+### Bug Fixes
+
+- Add more invalid characters for room aliases.
```

Comment:
I wish we could add links to the commit or the PR.

Reply:
If you add a `(#PR_NUMBER)` to your commit message they will be added: https://github.com/matrix-org/matrix-rust-sdk/blob/e1e1688a1ebd713f002c5c247e603943e60fb4d4/cliff.toml#L60.

Otherwise we have to set up the [github integration](https://git-cliff.org/docs/integration/github) for git-cliff.